mockwebserver/src/main/java/okhttp3/internal/tls/SslClient.java

```diff
@@ -79,6 +80,7 @@ public static synchronized SslClient localhost() {
     private final List<X509Certificate> certificates = new ArrayList<>();
     private KeyPair keyPair;
     private String keyStoreType = KeyStore.getDefaultType();
+    private SSLContext sslContext = null;
```

Comment:
nit: the `= null` has weird consequences and shouldn’t be done. The most surprising thing about it is that it shows up in the bytecode.

```
class X {
  String a = defaultValueOfA();
  String b = null;

  static String defaultValueOfA();
    b = "B";
    return "A";
  }
}
```

Regular programs don’t do this. But sometimes we accidentally write code that has side-effects like these. The `= null` doesn’t run when you’d expect it to, and so it yields surprising code.

My preference is to never explicitly assign fields to their natural default values (0, null, false).